content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Uninstall Pluggable Components from KubeSphere"
+keywords: "Installer, uninstall, KubeSphere, Kubernetes"
+description: "Learn how to uninstall each pluggable component in KubeSphere."
+linkTitle: "Uninstall Pluggable Components from KubeSphere"
+Weight: 16500
+---
+
+After you [enable the pluggable components of KubeSphere](../../../pluggable-components/), you can also uninstall them using the following commands. Please back up any necessary data before you uninstall these components.
+
+## Uninstall KubeSphere App Store
+
+```bash
+kubectl delete ns openpitrix-system
```

Comment:
In v3.1, we don't need to delete the ns anymore.

Reply:
So we just change `true` to `false` for `openpitrix.store.enabled` in the cc, then the App Store will be uninstalled?

Reply:
Yeap. BTW, the docs for v3.1 and v3.0 may be different. I guess we need to point that out.